R/news_handlers.R

```diff
@@ -7,13 +7,15 @@
 #' look for NEWs entries starting with \code{[version date]}
 #' @export
 #' @importFrom utils news
-#' @importFrom rprojroot find_root has_file
-vt_scrape_news <- function(pkg = rprojroot::find_root(criterion = rprojroot::has_file("DESCRIPTION"))){
+#' @importFrom devtools package_file
+vt_scrape_news <- function(pkg = devtools::package_file()){
+
 
-  if(is_package(pkg)){
+  if(suppressWarnings(require(pkg, character.only = TRUE, quietly = TRUE))){
```

Comment:
This loads the package, which I don't think is desirable since then we need to unload it. We could use `pkg %in% rownames(installed.packages())` to check to see if it is in the list of installed packages in the system. (or write a wrapper func to make it easier?)